karl/sandbox/exceptions.py

```diff
@@ -0,0 +1,18 @@
+class SandboxBaseException(Exception):
```

Comment:
See above review comment about setting messages in the constructor here (also applies for the children ofc)